tests/formparams/form_params_test.go

```diff
@@ -213,16 +219,21 @@ func TestFormParametersBreakUrlEncoding(t *testing.T) {
 		"veggie=%sc&fruit=apple\r\n" +
 		"\r\n")
 	resp, err := requesttesting.MakeRequest(context.Background(), postReq, func(req *http.Request) {
-		req.ParseForm()
+		err := req.ParseForm().Error()
+		wantErr := `invalid URL escape "%sc"`
+		if err != wantErr {
+			t.Errorf(`req.ParseForm(): want %v, got %v`, wantErr, err)
+		}
+
```

Comment:
Avoid comparing full error strings, as this makes the tests essentially "change detectors". In this case, it is fine to just check whether the error is non-nil.

Reply:
Resolved in [a180754](url)